src/app/feed/feed-card/feed-card.component.scss

```diff
@@ -109,8 +109,9 @@ a.post-link {
 	background-color: rgba(0, 0, 0, 0.5);
 
 	.overlay-content{
+		max-width: 90%;
+		height: 400px !important;
```

Comment:
It is required in order to get UI properly in some cases where the pixels are less than that defined limit.